src/main/java/com/exadel/discount/entity/City.java

```diff
@@ -0,0 +1,35 @@
+package com.exadel.discount.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "cities")
```

Comment:
Add here 
@EqualsAndHashCode(exclude = {"users"})
@ToString(exclude = {"users"})

Reply:
Done

Reply:
And now?